pages/404.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from "next/image";
+import { useRouter } from "next/router";
+import LandingFooter from "@/Components/LandingFooter";
+
+const NotFoundPage = () => {
+    const router = useRouter();
+  const navLinkStyle =
+    "font-archivo relative text-xl w-fit block after:block after:content-[''] after:absolute after:h-[3px] after:bg-black after:w-full after:scale-x-0 after:hover:scale-x-100 after:transition after:rounded-full after:duration-500 after:origin-center";
+  const handleBack = () => {
+    router.back();
+  };
+  const handleClick = () => {
```

Comment:
```suggestion
  const redirectToHome = () => {
```

The function name should be descriptive.